.github/workflows/build_and_test.yml

```diff
@@ -49,6 +66,7 @@ jobs:
     name: Windows
     runs-on: windows-2019
     timeout-minutes: 10
+    needs: browser
```

Comment:
Why do we need `browser` here?

Reply:
This stops this job from running until the browser tests succeed.